### PR TITLE
Rebuild & fix parse error

### PR DESCRIPTION
**Pull request by andychucs**

### CODE REVIEW

File: .github/workflows/update.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      run: |
+        python src/main.py kcQuests -d 1
+    - name: Copy file
+      run: |
+        cp quests-scn.json dist/quests-scn.json
+        cp quests-scn-new.json dist/quests-scn-new.json
+    # Runs a set of commands using the runners shell
+    - name: Commit changes
+      uses: EndBug/add-and-commit@v9
+      with:
```
Comment (lawvs) [inline .github/workflows/update.yaml:40]: I don't think it's a good idea to commit changes when submitting a pull request to the main branch.

```suggestion
      if: github.ref == 'refs/heads/main'
      with:
```

Comment (andychucs) [reply .github/workflows/update.yaml:40]: https://github.com/kcwikizh/kcQuests/pull/9/commits/a02119e6d77a14bd49fd6d27ba5858ee8d6d0f2d

File: quests-scn-new.json [hunk 2]
```diff
@@ -11,136 +11,138 @@
     ]
   },
   "1002": {
-    "code": "2310B5 ",
+    "code": "2310B5",
     "desc": "以含「鹈来」「稻木」「朝潮」「夕立」「野分」「滨波」「早波」「阿武隈」中任意2艘的精锐舰队歼灭镇守府近海、东部奥廖尔海、加斯加达马岛、中部北海域孔雀岛近海的敌方战力！",
-    "memo": "奖励:以下奖励三选一,大发动艇×4 ,装甲艇(AB艇)★+4 ,一式陆攻三四型★+4,以下奖励三选一,试制东海★+2,战斗详报×1,南瓜×3",
+    "memo": " 以下奖励三选一 大发动艇 ×4  装甲艇(AB艇)  green|★+4   一式陆攻三四型  green|★+4 以下奖励三选一 试制东海  green|★+2  战斗详报 ×1 南瓜 ×3 ",
```
Comment (andychucs) [inline quests-scn-new.json]: waiting to fix

**Comment by andychucs** [comment]
need check why wiki data rs/0-26.txt not have comment with id

**Comment by TrickLin** [comment]
There seems to be some missing information, such as pre-quest for Fy7，F101 in dist/quests-scn.json 
and please add the `奖励:` prefix to ”memo“ field

File: src/kcwiki/quest.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    @classmethod
+    def from_wt_template(cls, template, items):
+        quest = cls()
+        quest.id = template.comments[0].contents.strip()
+        for arg in template.arguments:
+            name = arg.name.strip()
+            value = arg.value.strip()
+            if name == '编号':
+                quest.code = value
+            if name.startswith('前置') and value:
```
Comment (andychucs) [inline src/kcwiki/quest.py:52]: `if '前置' in name and value:`

File: src/kcwiki/quest.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+    def to_dict(self):
+        return {
+            self.id: self.__dict__
+        }
+
+    def to_dict_without_id(self):
+        return {
+            self.id: {
+                i: self.__dict__[i] for i in self.__dict__.keys() if i != 'id'
```
Comment (andychucs) [inline src/kcwiki/quest.py]:
```suggestion
                i: self.__dict__[i] if i != 'memo' else f'奖励:{self.__dict__[i]}' for i in self.__dict__.keys() if i != 'id'
```

File: src/kcwiki/quest.py (hunk 3 above)
Comment (andychucs) [inline src/kcwiki/quest.py]:
```suggestion
            if '前置' in name and value:
```

**Pull request merged**